packages/unmock-node/src/loaders/fs-service-def-loader.ts

```diff
@@ -0,0 +1,89 @@
+import debug from "debug";
+import fs from "fs";
+import path from "path";
+import { IServiceDef, IServiceDefLoader } from "unmock-core";
+
+const DEFAULT_SERVICE_SUBDIRECTORY = "__unmock__";
+
+export interface IFsServiceDefLoaderOptions {
```

Comment:
Could create an `interfaces.ts` and `constants.ts` in this folder as well, and move those there.

Reply:
Hmm, they will then each contain only a single definition?

Reply:
True, but if it's exported and used elsewhere it might be cleaner. If this are the only constants/interfaces you think will be used in `loaders` in general than I don't think it's needed. I thought there'll be more implementations coming, so it made sense (otherwise IMO it doesn't require its own folder)